frontend/src/services/protectedemail.service.ts

```diff
@@ -0,0 +1,30 @@
+export interface CsvStatus {
+  csvFilename?: string
+  csvError?: string
```

Comment:
Hmmm i'm not sure if the `csvError` should be a string or an array of strings because there could be multiple rows with missing params. @miazima what do you think? Should we display all the errors or allow them to download the errors? 

We could stick with a generic string first for now, and open an issue so that the scope of this work don't become too big.